zerver/openapi/zulip.yaml

```diff
@@ -6397,10 +6415,38 @@ paths:
                         <td><code>wildcard_mentioned</code></td>
                         <td>
                             Whether this message contained
-                            <a href="/help/mention-a-user-or-group#mention-everyone-on-a-stream">wildcard mention</a>
+                            <a href="/help/mention-a-user-or-group#mention-everyone-on-a-stream">stream wildcard mention</a>
                             like @**all**. Cannot be changed by the user directly, but
                             can change if the message is edited to add/remove
-                            a wildcard mention.
+                            a stream wildcard mention.
+
+                            **Changes**: In Zulip 8.0 (feature level 215), this flag was deprecated,
+                            replaced with `stream_wildcard_mentioned` and `topic_wildcard_mentioned`.
+                            It exists for backwards compatibility and will be removed once clients
+                            migrate to use the new flags.
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><code>stream_wildcard_mentioned</code></td>
+                        <td>
+                            Whether this message contained
+                            <a href="/help/mention-a-user-or-group#mention-everyone-on-a-stream">stream wildcard mention</a>
+                            like @**all**. Cannot be changed by the user directly, but
+                            can change if the message is edited to add/remove
+                            a stream wildcard mention.
+
+                            **Changes**: New in Zulip 8.0 (feature level 215).
+                        </td>
+                    </tr>
+                    <tr>
+                        <td><code>topic_wildcard_mentioned</code></td>
+                        <td>
+                            Whether this message contained topic wildcard mention i.e.
+                            @**topic**. Cannot be changed by the user directly, but
```

Comment:
"Whether this message contained topic wildcard mention (`@**topic**`)" is probably clearer than the `i.e`.

Reply:
Done